simcon_project/templates/create_assignment.html

```diff
@@ -0,0 +1,268 @@
+          var tUsername = document.createElement('td');
+          var tKey = document.createElement('td');
+          //Add the data submitted by the form to each cell
+          tUsername.innerHTML = name;
+          tKey.innerHTML = key;
+          //Add and delete hyperlinks
+          var tDel = document.createElement('td');
+          tDel.innerHTML = '<a href="javascript:;">Delete</a>';
+          //Perform delete table row operation
+          var oA = tDel.children[0];
+          oA.onclick = function(){
+              tBody.removeChild(this.parentNode.parentNode);
+              var em=this.parentNode.parentNode.cells[1].innerHTML;
+              //student
+              if(id==1) {
+                  var index = addedStu.indexOf(em);
+                  console.log(em + "  " + index);
+                  if (index > -1) {
+                      addedStu.splice(index, 1);
+                  }
+              }
+              //template
+              else if(id==2){
+                  var name=this.parentNode.parentNode.cells[0].innerHTML;
+                  var tempKey=name+"--"+em;
+                  var index = addedTmp.indexOf(tempKey);
+                  console.log(tempKey + "  " + index);
+                  if (index > -1) {
+                      addedTmp.splice(index, 1);
+                  }
+              }
+              //label
+              else{
+                  var index = addedLabel.indexOf(em);
+                  console.log(em + "  " + index);
+                  if (index > -1) {
+                      addedLabel.splice(index, 1);
+                  }
+              }
+          };
+          //Add cells and rows to the table
+          tr.appendChild(tUsername);
+          tr.appendChild(tKey);
+          tr.appendChild(tDel);
+          tBody.appendChild(tr);
+      }
+
+      function add_stu(tbl, stu){
+          var find=false;   //Record find the corresponding student or not
+          for(var i=0;i<students.length;i++){
+              var stuTmp=students[i];
+              if(stuTmp.email==stu){
+                  find=true;
+                  if(stuTmp.is_active!=1){
+                      window.alert("Add failed. This user is a not an active user");
+                  }
+                  else{
+                      //Determine whether this student is already in the table.
+                      if(addedStu.indexOf(stu)>-1){
+                          window.alert("Adding failed, the student is already in the table.");
+                      }
+                      else {
+                          var name = stuTmp.first_name + " " + stuTmp.last_name;
+                          add2table(tbl, name, stu, 1);
+                          addedStu.push(stu);
+                      }
+                  }
+              }
+          }
+          if(find==false){
+              window.alert("Cannot find: "+stu);
+          }
+      }
+
+      function add_temp(tbl, name){
```

Comment:
When I create an assignment, the template I chose doesn't end up getting included along with the assignment.

Reply:
> When I create an assignment, the template I chose doesn't end up getting included along with the assignment.

Can you describe it dependently? I'm not sure how to improve it.

Reply:
My bad, I must've been working with an old version. Ignore.